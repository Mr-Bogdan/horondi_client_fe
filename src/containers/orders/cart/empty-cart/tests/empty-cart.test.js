import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import * as reactRedux from 'react-redux';

import EmptyCart from '../index';

configure({ adapter: new Adapter() });

let wrapper;

function spyOnSelector(language) {
  jest.spyOn(reactRedux, 'useSelector').mockImplementation((cb) =>
    cb({
      Language: { language }
    })
  );
}

describe('Empty cart component tests', () => {
  it('should match snapshot', () => {
    spyOnSelector(0);

    wrapper = shallow(<EmptyCart />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should cover other branches', () => {
    spyOnSelector(1);

    wrapper = shallow(<EmptyCart />);
  });
});
