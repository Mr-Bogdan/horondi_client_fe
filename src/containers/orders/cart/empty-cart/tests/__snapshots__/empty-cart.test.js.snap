// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty cart component tests should match snapshot 1`] = `
<EmptyOrder
  buttonTitle="cart.empty"
  name="empty-cart"
  title="cart.emptyCart"
/>
`;
